solutionbox/inception/datalab_solutions/inception/_local.py

```diff
@@ -73,7 +78,62 @@ def predict(self, model_dir, image_files, labels_file):
           feed_dict[inputs['key']].append(str(ii))
       predictions, scores = sess.run([outputs['prediction'], outputs['scores']],
                                      feed_dict=feed_dict)
-    
     labels_and_scores = [(labels[predicted_index], class_scores[predicted_index])
                          for predicted_index, class_scores in zip(predictions, scores)]
     return labels_and_scores
+
+
+  def batch_predict(self, model_dir, input_csv, labels_file, output_file, output_bq_table):
```

Comment:
I think there is too much code in the prediction functions in this file. The work done would be pulled out into a _local_predictions.py file. I think some code in local predict/batch_predict can be reused. Your call.